modules/microshift-rhoai-query-model.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * microshift_ai/microshift-rhoai.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhoai-query-model_{context}"]
+= Querying your AI model
+
+Make an inference request against the AI model server that is using the `ovms-resnet50` model.
+
+.Prerequisites
+
+* The {microshift-short} cluster is running.
+* You configured the model-serving runtime.
+* You uploaded your AI model to {microshift-short}.
+
+.Procedure
+
+. Make an inference request against the model server that is using the `ovms-resnet50` model by running the following command:
```

Comment:
Since this is a single step procedure, this item should be unordered instead:
```suggestion
* Make an inference request against the model server that is using the `ovms-resnet50` model by running the following command:
```